src/applications/facility-locator/utils/selectors.js

```diff
@@ -1,4 +1,5 @@
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
 export const facilityLocatorShowCommunityCares = state =>
-  toggleValues(state).facilityLocatorShowCommunityCares;
+  toggleValues(state)[FEATURE_FLAG_NAMES.facilityLocatorShowCommunityCares];
```

Comment:
I don't like the bracket notation but this _is_ safer. I'm open to other solutions for this. We could just build on the site-wide feature-toggle selector and make a selector for each feature flag but it feels like actual feature flag names should be managed at the app level